packages/gradient-block/src/components/SquareBadgesRow.tsx

```diff
@@ -23,7 +23,9 @@ export const SquareBadgesRow = ({ blockWidth, gradientColors, gradientOrientatio
     };
 
     useEffect(() => {
-        prepareGradients();
+        if (gradientColors.length > 0) {
```

Comment:
maybe we could display a info text like `Please note that the gradient block requires a minimum of two colors to be selected in order to function properly.` Wdyt?

Reply:
Yeah i think that would be great, any idea where to put it?